tests/cephfs/cephfs_nfs/test_cephfs_nfs_ha_active_power_off.py

```diff
@@ -35,7 +35,7 @@ def run(ceph_cluster, **kw):
     8. Cleanup
     """
     try:
-        tc = "CEPH-83575196"
+        tc = "CEPH-83575195"
```

Comment:
The test in polarion is already marked Automated ?? Is the polarion ID for the test correct?

Reply:
This automated Neha. But by mistake it had import pdb and pdb.set_trace() so removed it as part of this PR